apps/api/src/app/change/usecases/create-change/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from './create-change.command';
+export * from './create-change.usecase';
```

Comment:
Following the same pattern of the use case we apply everywhere, I moved to dedicated folder to easier export.

Reply:
Nice!